v1-api-conversion/README.md

```diff
@@ -0,0 +1,372 @@
+# Strimzi `v1` API Conversion Tool
+
+Strimzi’s Custom Resource Definitions (CRDs) have been updated to remove deprecated fields in the new `v1` schema. 
+To support this transition, we've introduced a CLI-based API conversion tool. 
+
+The tool supports two main operations:
+* Converting existing custom resources to the `v1` schema before upgrading the CRDs:
+    * From YAML files (`convert-file`)
+    * Directly in your Kubernetes cluster (`convert-resource`)
+* Upgrading CRDs by replacing existing Strimzi CRDs with their `v1` equivalents (`crd-upgrade`)
+
+You can list the tool's commands using `help`:
+
+```
+> bin/api-conversion.sh help
```

Comment:
I guess that starting from here, all the examples should have `v1-api-conversion.sh` and not `api-conversion.sh`. Or am I missing anything here?